src/terra-dev-site/test/modal-manager/CustomDismissCheck.test.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import classNames from 'classnames/bind';
+
+import ModalManager from '../../../modal-manager';
+
+import DisclosureComponent from './DisclosureComponent';
+import styles from './ModalManagerCommon.test.module.scss';
+
+const cx = classNames.bind(styles);
+
+const CustomDismissCheck = () => (
+  <div role="main" className={cx('content-wrapper')}>
```

Comment:
Do we need the test wrapper?

Reply:
As it turns out, I don't think so? Some prior tests had these, I removed them there and in these new tests.